src/routes/auth.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+"""Contains the autorization handlers."""
+import time
+
+from jose import JWTError, jwt
+from werkzeug.exceptions import Unauthorized
+from src.config import SECRET_CLIENT_API_TOKEN
+
+
+def decode_token(token):
```

Comment:
Mostly for Auth story you might be still working on it, but just pointing out incase if you missed
- API restriction with some decorator that validate request bearer token with DB data    
- generate_token should store generated token into DB and request can be authenticated with SECRET_TOKEN_GENERATION
- decode_token should not be a route, this might be you will use with decorator to authenticate API request

Reply:
@rajusem it is not a route per se, it is marked as a bearer token. Let me schedule a call to discuss this workflow once.